plugin/src/leiningen/cljs_lambda.clj

```diff
@@ -118,12 +119,17 @@
   "Write a zip file suitable for Lambda deployment"
   [{{:keys [cljs-build cljs-build-id functions resource-dirs env] :as opts} :cljs-lambda
     :as project}]
+  (if (or (opts :managed-deps) (-> opts :keyword-args :managed-deps))
+    (log :verbose "Note: You set the :managed-deps options to true, so dependencies won't be handled automatically.")
+    (if (.exists (io/as-file "package.json"))
+      (do (log :verbose "Your project already has a project.json file. Please remove it and use :npm {:dependencies [...]} option instead.")
```

Comment:
Thanks a bunch for this.  This expr may be clearer as `(main/abort "Your project...")` w/out the `(do)`.  What do you think?